roles/usegalaxy-eu.bashrc/tasks/bashrc_tasks.yml

```diff
@@ -0,0 +1,119 @@
+      stat:
+        path: "{{ item.uhome }}/.bashrc"
+      register: bashrc_stat_out
+
+    - name: Copy default bashrc when not existing
+      copy:
+        src: /etc/skel/.bashrc
+        dest: "{{ item.uhome }}/.bashrc"
+        remote_src: yes
+        mode: 0640
+        owner: "{{ item.uname }}"
+        group: "{{ item.gname }}"
+      when: not bashrc_stat_out.stat.exists
+
+    - name: Check for bashprofile
+      stat:
+        path: "{{ item.uhome }}/.bash_profile"
+      register: bashprofile_stat_out
+
+    - name: Check for profile
+      stat:
+        path: "{{ item.uhome }}/.profile"
+      register: profile_stat_out
+
+    - name: Copy default bashprofile when not existing
+      copy:
+        src: /etc/skel/.bash_profile
+        dest: "{{ item.uhome }}/.bash_profile"
+        remote_src: yes
+        mode: 0640
+        owner: "{{ item.uname }}"
+        group: "{{ item.gname }}"
+      when: not bashprofile_stat_out.stat.exists and not profile_stat_out.stat.exists
+
+    - name: Copy galaxy_jwd python script
+      copy:
+        src: galaxy_jwd.py
+        dest: /usr/local/bin/galaxy_jwd
+        mode: 0755
+        owner: galaxy
+        group: galaxy
+
+    - name: Insert some aliases and functions
+      blockinfile:
+        path: "{{ item.uhome }}/.bashrc"
+        marker: "# {mark} ANSIBLE MANAGED BLOCK"
+        content: |
+          # User specific aliases and functions
+          function change_to_wd() {
+          USAGE="Please provide a Galaxy job ID or a Condor job ID"
+          if (( $# == 0 )); then
+                  echo $USAGE
+                  return 0;
+          fi
+          for i in "$@"; do
+                          if [[ "$i" = --help || "$i" = -h ]]; then
+                                  echo $USAGE
+                                  return 0;
+                          fi
+          done
+          JID=$1
+          WD=$(dirname `condor_q -autoformat Cmd ClusterId | grep ${JID} | cut -f1 -d' '` || dirname `condor_history -autoformat Cmd ClusterId | grep ${JID} | cut -f1 -d' '` || find "{{ galaxy_config['galaxy']['job_working_directory'] }}""/0"${JID:0:2}"/"${JID:2:3} -maxdepth 1 -type d -name ${JID})
+          cd $WD
+          }
+
+          # Uses the /usr/local/bin/galaxy_jwd python script to change to the job working directory
+          function change_to_jwd() {
+          USAGE="Please provide a Galaxy job ID"
+          if (( $# == 0 )); then
+              echo $USAGE
+              return 0;
+          fi
+
+          JID=$1
+          JWD=$(python /usr/local/bin/galaxy_jwd get_jwd $JID)
+          cd $JWD
+          }
+
+          alias gl='journalctl  -f -u galaxy-*'
+          alias glg='journalctl -fu galaxy-gunicorn@* | grep -v -e "/api/upload/hooks" -e "/history/current_history_json"'
+          alias glh='journalctl -f -u galaxy-handler@*'
+          alias glw='journalctl -f -u galaxy-workflow-scheduler@*'
+          alias glc='journalctl -fu galaxy-celery@*'
```

Comment:
celery is not on the head nodes anymore, or is this role also used on the celery host?

Reply:
No, the role is not used in the `celery.yml` playbook. I think it's legacy code. I will remove it and add a new commit.

Reply:
Updated: https://github.com/usegalaxy-eu/infrastructure-playbook/pull/610/commits/229c0cb0c49448f2b9843d5b1948a41186c11da5